### PR TITLE
cli: clarify program source flag description

### PR DESCRIPTION
**Pull request by kehiy**
## Summary
hello Algorand team, I add a new desc to the `clerk` CLI help command to show the -F and -P flags should point to a file.

# Issues
 - #1711

### CODE REVIEW

**Comment by kehiy** [comment]
@zlangley @winder

File: cmd/goal/clerk.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func init() {
 	sendCmd.Flags().Uint64VarP(&amount, "amount", "a", 0, "The amount to be transferred (required), in microAlgos")
 	sendCmd.Flags().StringVarP(&closeToAddress, "close-to", "c", "", "Close account and send remainder to this address")
 	sendCmd.Flags().StringVar(&rekeyToAddress, "rekey-to", "", "Rekey account to the given spending key/address. (Future transactions from this account will need to be signed with the new key.)")
-	sendCmd.Flags().StringVarP(&programSource, "from-program", "F", "", "Program source to use as account logic")
+	sendCmd.Flags().StringVarP(&programSource, "from-program", "F", "", "Program source to use as account logic, should point to a file")
```
Comment (winder) [inline cmd/goal/clerk.go:101]: Would this shorter description have clarified the command for you?
```suggestion
	sendCmd.Flags().StringVarP(&programSource, "from-program", "F", "", "Program source file to use as account logic")
```

**Comment by kehiy** [comment]
@winder I fixed this, you can merge it now

**Comment by kehiy** [comment]
@winder I fixed this, you can merge it now

**Review by winder — APPROVED** [review]

**Comment by winder** [comment]
Thanks!

**Review by jannotti — APPROVED** [review]

**Comment by jannotti** [comment]
I'd rather see the whole `s` defined in one piece using backticks to avoid all the double quotes, the escaping, and concatenation altogether.

**Pull request merged**